### PR TITLE
ArangoDB module was added

### PR DESCRIPTION
**Pull request by ltwlf**
This was rather simple :)

### CODE REVIEW

File: src/modules/arangodb/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# ArangoDB
+
+ArangoDB is an open source frienly multi-model database.
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:3]: typo friendly -> friendly?

File: src/modules/arangodb/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Examples
+
+Jest test to verify the connection
+
+```typescript
+import { Database } from "arangojs";
+import { ArangoDBContainer, StartedArangoContainer } from "./arangodb-container";
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:11]: `from "testcontainers"`

File: src/modules/arangodb/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    const result = await db.query({
+      query: "RETURN @value",
+      bindVars: { value },
+    });
+    const returnValue = await result.next();
+    expect(returnValue).toBe(value);
+  });
+});
+
+```
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:46]: empty code block?

File: src/modules/arangodb/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# ArangoDB
+
+ArangoDB is an open source frienly multi-model database.
+
+## Examples
+
+Jest test to verify the connection
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:7]: i wonder if we should include the full test example, or just the different ways in which the container can be used like 
```javascript
new ArandoDBContainer().start()
```
the two reasons i mention it is that whoever is looking at the README will see the test file right there, and secondly this README is a copy of the test file, so why worry about keeping two things in sync?

File: src/modules/arangodb/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  });
+});
+
+```
+
+
+```
+## Contact
+You miss something? 
+Create an issue or write me on [twitter](https://twitter.com/ltwlf)
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:52]: you don't think it'd be better to raise the issue on GH? perhaps mentioning your user handle? that way other people could also fix it :slightly_smiling_face:

File: src/modules/neo4j/README.md [hunk 6]
```diff
@@ -41,8 +64,14 @@ Install APOC plugin:
...
 ```typescript
-container = await new Neo4jContainer().withApoc().start();
-  ```
+container = await new Neo4jContainer()
+  .withApoc()
+  .withTtl(5)
+  .withPassword('super_secret')
+  .start();
+```
+## Contact
```
Comment (cristianrgreco) [inline src/modules/neo4j/README.md:75]: same comment as above

File: src/modules/neo4j/neo4j-container.ts [hunk 7]
```diff
@@ -30,6 +30,10 @@ export class Neo4jContainer extends GenericContainer {
     );
   }
 
+  public withTtl(schedule = 5) {
```
Comment (cristianrgreco) [inline src/modules/neo4j/neo4j-container.ts:33]: was this not already merged into master?

**Comment by cristianrgreco** [comment]
thanks @ltwlf, just a few questions/comments :slightly_smiling_face:

File: src/modules/arangodb/README.md (hunk 2 above)
Comment (ltwlf) [reply src/modules/arangodb/README.md]: Linked the jest test instead

File: src/modules/arangodb/README.md (hunk 3 above)
Comment (ltwlf) [reply src/modules/arangodb/README.md]: fixex

File: src/modules/arangodb/README.md (hunk 4 above)
Comment (ltwlf) [reply src/modules/arangodb/README.md]: You are right. I think it is best to link the test. ArangoDB is well documented. I linked the documentation too.

File: src/modules/neo4j/neo4j-container.ts (hunk 7 above)
Comment (ltwlf) [reply src/modules/neo4j/neo4j-container.ts:33]: Yes, this is weird. I can't see any difference here. Do you know how to remove this from the pull? When I checkout the file from the base branch it doesn't show any modificationn

File: src/modules/arangodb/README.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+# ArangoDB
+
+[[ArangoDB](https://www.arangodb.com/) is an open source friendly multi-model database.
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:3]: extra `[]`

File: src/modules/arangodb/README.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+
+You can find the documentation of the [ArangoDB JavaScript](https://www.arangodb.com/docs/stable/drivers/js.html) driver here. 
+
+## Examples
+
+Check out the [jest test](arangodb-container.test) to see how to start the container and connect the ArangoDB with the JavaScript driver.
```
Comment (cristianrgreco) [inline src/modules/arangodb/README.md:9]: `arangodb-container.test.ts`?

File: src/modules/neo4j/README.md [hunk 10]
```diff
@@ -1,38 +1,61 @@
...
-const personName = "Chris";
+afterEach(async () => {
+  await session?.close()
+  await driver?.close()
+  await container?.stop()
+})
 
-try {
-  const result = await session.run("CREATE (a:Person {name: $name}) RETURN a", { name: personName });
+describe('neo4j', () => {
```
Comment (cristianrgreco) [inline src/modules/neo4j/README.md:32]: is it intentional that the beforeEach/afterEach is outside of the describe here but not in the arango-db example?

File: src/modules/neo4j/neo4j-container.ts (hunk 7 above)
Comment (cristianrgreco) [reply src/modules/neo4j/neo4j-container.ts:33]: seems there's a conflict here now

**Comment by cristianrgreco** [comment]
just a couple comments about the README, once addressed lgtm 👍

File: src/modules/neo4j/README.md (hunk 10 above)
Comment (ltwlf) [reply src/modules/neo4j/README.md:32]: Same result when there is only one test suite. I'll move it in the global scope. Usually I would create a new test file when I've to switch a container setup.

File: src/modules/neo4j/neo4j-container.ts (hunk 7 above)
Comment (ltwlf) [reply src/modules/neo4j/neo4j-container.ts]: I don't get a conflict here

**Comment by cristianrgreco** [comment]
great stuff! let me know if it's ready to release

**Comment by ltwlf** [comment]
Fine for me :)

**Comment by cristianrgreco** [comment]
released in 3.4.0 👍

**Pull request merged**